optaplanner-benchmark/src/main/java/org/optaplanner/swing/impl/SwingUncaughtExceptionHandler.java

```diff
@@ -49,13 +51,16 @@ public static void register() {
 
     @Override
     public void uncaughtException(Thread t, Throwable e) {
+        StringWriter sw = new StringWriter();
```

Comment:
I 'll rename variable to stringWriter after merging.
(only well known variables can be very short, such as Object o, Exception e, int i, etc)